app/mailers/spree/order_mailer_decorator.rb

```diff
@@ -3,43 +3,42 @@
   helper CheckoutHelper
   helper SpreeCurrencyHelper
 
-  def cancel_email(order, resend = false)
-    @order = find_order(order)
-    subject = (resend ? "[#{t(:resend).upcase}] " : '')
-    subject += "#{Spree::Config[:site_name]} #{t('order_mailer.cancel_email.subject')} ##{@order.number}"
-    mail(to: @order.email, from: from_address, subject: subject)
-  end
-
-  def confirm_email_for_customer(order, resend = false)
-    find_order(order) # Finds an order instance from an id
-    subject = (resend ? "[#{t(:resend).upcase}] " : '')
-    subject += "#{Spree::Config[:site_name]} #{t('order_mailer.confirm_email.subject')} ##{@order.number}"
+  def confirm_email_for_customer(order_or_order_id, resend = false)
+    @order = find_order(order_or_order_id)
+    subject = build_subject("#{t('order_mailer.confirm_email.subject')}", resend)
```

Comment:
You don't need the string interpolation any more:
```ruby
subject = build_subject(t('order_mailer.confirm_email.subject'), resend)
```
There are a couple more cases in this file.

Reply:
ah, true, well spotted! Fixed.